### PR TITLE
chore(testing): bump protractor to version 1.6.0

### PR DESCRIPTION
**Pull request by juliemr**
This will also be cherry-picked to 1.3 and 1.2

### CODE REVIEW

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -1528,7 +1528,7 @@
               "dependencies": {
                 "fsevents": {
                   "version": "0.2.1",
-                  "from": "git://github.com/pipobscure/fsevents#7dcdf9fa3f8956610fd6f69f72c67bace2de7138",
+                  "from": "fsevents@git://github.com/pipobscure/fsevents#7dcdf9fa3f8956610fd6f69f72c67bace2de7138",
```
Comment (lgalfaso) [inline npm-shrinkwrap.json]: ?!

Comment (juliemr) [inline npm-shrinkwrap.json]: I don't know, this keeps changing when I run npm-shrinkwrap and in the past I've been told it doesn't matter. Happy to revert this part though.

**Comment by lgalfaso** [comment]
please revert the `fsevents` line, otherwise LGTM

**Comment by juliemr** [comment]
Done, merged as eb6cb785df63bdd06c0bccee39ea437baa8f628e

Merged to 1.3.x as eb6cb785df63bdd06c0bccee39ea437baa8f628e

**Pull request closed without merging**